src/google/cloud/ndb/model.py

```diff
@@ -593,7 +593,7 @@ def __init__(
         choices=None,
         validator=None,
         verbose_name=None,
-        write_empty_list=None
+        write_empty_list=None,
```

Comment:
Um, blacken step adds this comma or else the lint session fails, but it seems to cause a syntax error on python 3.5 unit tests. The code didn't have the trailing commas before. Am I missing some way to ignore this blacken change without failing the build?

Reply:
I've been playing with this. It looks like the "*," in the argument list affects it. Syntax is fine if you take that out, but leave in the trailing comma. Unfortunately, that means you can't make truly keyword only args without just using **kwargs, which you then have to validate by hand in the function definition, which isn't awesome.

There are also some f"" strings in here that aren't python 3.5 compatible. 

@crwilcox @andrewsg Do we need python 3.5? Python 3.6 and 3.7 seem much better behaved in this regard.

Reply:
Ah, I see now the "*," was added by Danny in porting. Python 2 doesn't support that syntax. Since we're not losing anything porting from 2 to 3 by not using that syntax, I'd say let's just get rid of any instances of this and make the function signatures look like the Legacy NDB versions.

Reply:
Thanks for the input. Just removing those causes errors in all three builds, but I'll find a solution. I do believe it's a valid question whether we should still support 3.5, though.